src/components/Pricing/PlanComparison/index.tsx

```diff
@@ -83,18 +83,9 @@ const sections = [
                 },
             },
             {
-                name: 'Automatic updates',
+                name: 'Updates on schedule',
```

Comment:
I don't think this is as clear as "automatic updates" or "continuous updates", context on this change?

Reply:
I only wanted to update the Zapier box here, but then I saw this and it's pretty much false, as we don't have an autoupdate mechanism for self-hosted. We try to make updates smooth, but AFAIK we don't even fully manage customers' instances anymore, so generally as a customer it's your task to update.

Reply:
Agreed, I thought this is meant to convey that you don't need to worry about updates when on Cloud, so I think this would only be true for Cloud but do think the original header is clearer. That being said, not sure how good of a job we're doing communicating this here, might be worth just getting rid of this line altogether.